testsuite/tests/regression/pr6608/Makefile

```diff
@@ -1,11 +1,15 @@
-default: test
+default: test_1 test_2
 
-test:
-	@printf " ... testing 'pr6608.reference':"
-	@$(OCAMLC) -c pr6608.ml 2> pr6608.result || true
-	@$(DIFF) pr6608.result pr6608.reference >/dev/null \
+test_1:
+	@printf " ... testing 'pr6608_1.reference':"
+	@$(OCAMLC) -c pr6608_1.ml 2> pr6608_1.result || true
+	@$(DIFF) pr6608_1.result pr6608_1.reference >/dev/null \
 	    && echo " => passed" || echo " => failed"
 
+test_2:
+	@printf " ... testing 'pr6608_2.reference':"
+	@$(OCAMLC) -c pr6608_2.ml && echo " => passed" || echo " => failed"
+
```

Comment:
Instead of writing your own testing Makefile, you could simply reuse the `toplevel` one that runs both of them through the toplevel, as a way to check either typing errors or the results of type inference. `testsuite/tests/typing-unboxed` would be an example of that, the Makefile is very simple, and you just have to create a `.reference` file for each `.ml` file in the test directory.